STMicroelectronics/STM32L4_L4+/app/az_ulib_dm/inc/_az_ulib_dm_blob_ustream_interface.h

```diff
@@ -0,0 +1,52 @@
+#ifndef AZ_ULIB_DM_BLOB_USTREAM_INTERFACE_H
+#define AZ_ULIB_DM_BLOB_USTREAM_INTERFACE_H
+
+#include "az_ulib_result.h"
+#include "az_ulib_ustream.h"
+#include "azure/az_core.h"
+#include "nx_api.h"
+#include "nx_web_http_client.h"
+#include "nx_wifi.h"
+#include "nxd_dns.h"
+#include "stm_networking.h"
+#include "wifi.h"
+
+
+#ifndef __cplusplus
+#include <stdint.h>
+#else
+#include <cstdint>
+#endif /* __cplusplus */
+
+#define DCF_WAIT_TIME 600
```

Comment:
might want to change the name to something less generic, something like BLOB_TIMEOUT

Reply:
Will change to something in line with the file naming, but it's technically not based on any timeout values. It's the amount of time we wait for all of the NX APIs before we return an error.